### PR TITLE
add hostname to loki label

### PR DESCRIPTION
**Pull request by bibace**
add hostname to loki label

### CODE REVIEW

File: remotelog/loki.go [hunk 1]
```diff
@@ -124,6 +125,9 @@ func (l *Loki) start(hCh chan *decoder.HEP) {
 				l.entry.labels["protocol"] = model.LabelValue(protocol)
 			}
 
+			hostname, _ := os.Hostname()
```
Comment (negbie) [inline remotelog/loki.go:128]: 1) It's best practice in Go to always check the error.
2) The os.Hostname() call ist expensive and just needs to be done once so move it out of the for loop.

Comment (bibace) [reply remotelog/loki.go]: Thanks @negbie,  updated

**Comment by negbie** [comment]
LGTM!

**Review by lmangani — APPROVED** [review]

**Review by negbie — APPROVED** [review]

**Comment by negbie** [comment]
Thank you @bibace

**Comment by adubovikov** [comment]
remotelog/loki.go:135:31: cannot use hostname (type string) as type model.LabelValue in assignment

**Comment by adubovikov** [comment]
should be:
```
                        l.entry.labels["hostname"] = model.LabelValue(hostname)

```

please fix :-)

**Comment by bibace** [comment]
sure, I am checking the code. will fix it asap

**Pull request merged**